2.0/woocommerce-easypay_cc_2/public/visa-fwd-2.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+if (!isset($_GET["e"])
+    || !isset($_GET["r"])
+    || !isset($_GET["v"])
+    || !isset($_GET["s"])
+) {
+    die("Error! Missing parameters!");
+}
+
+$explodedFilePath = explode('wp-content', __FILE__);
+$wpLoadFilePath = reset($explodedFilePath) . '/wp-load.php';
+
+require_once $wpLoadFilePath;
+
+global $woocommerce, $wpdb;
+
+if (!class_exists('WC_Gateway_Easypay_CC')) {
+    include realpath(plugin_dir_path(__FILE__)) . DIRECTORY_SEPARATOR
+        . '..' . DIRECTORY_SEPARATOR . 'includes' . DIRECTORY_SEPARATOR
+        . 'wc-gateway-easypay-cc.php';
+}
+
+$wcep = new WC_Gateway_Easypay_CC();
+$order = new WC_Order($_GET['t_key']);
```

Comment:
You should validate $_GET['t_key'] before try to get order. Just because woocommerce usually use order id as order identification.

Reply:
Done :)